examples/tts/conf/mixer-tts.yaml

```diff
@@ -27,8 +27,9 @@ pitch_loss_scale: 0.1
 durs_loss_scale: 0.1
 mel_loss_scale: 1.0
 
-phoneme_dict_path: "scripts/tts_dataset_files/cmudict-0.7b-030921"
+phoneme_dict_path: "scripts/tts_dataset_files/cmudict-0.7b"
```

Comment:
Let's add this back to NeMo. For now, we can use base 07 cmudict. Or whatever we decide in the Wednesday meeting

Reply:
Fixed.